src/js/components/login.js

```diff
@@ -6,10 +6,13 @@
 */
 import classnames from 'classnames';
 import FluxComponent from 'flummox/component';
+import connectToStores from 'flummox/connect';
 import fluxMixin from 'flummox/mixin';
 import React from 'react';
 import Url from 'urlgray';
 
+import LoginForm from './handlers/loginform';
```

Comment:
I might prefix all fxa things with `fxa`, to make a distinction between our login handling and theirs.